index.js

```diff
@@ -11,6 +11,32 @@ const hexToRGB = (hexColor) => {
   return [R, G, B];
 };
 
+const rgbToRGB = (rgbColor) => {
```

Comment:
I understand why you called this func rgbToRGB.
But it appears to be a little bit confusing name.
Maybe give it a liitle bit more extensive name.
Like
`rgbInputToRGBNumbers` (I think this function basically doing just that - extracting numbers for calculation)
or maybe some other name

Reply:
Oh yeah, I do have an issue with naming stuff, so please feel free to correct me anytime.
It is confusing. I just wanted to stay consistent with the naming of the other functions: `hexToRGB`, `hslToRGB`.
So do you think that changing `rgbToRGB` to `rgbInputToRGBNumbers` would be fine without changing the other two.
Maybe you have another name for the `rgbaToCloseRGB` function, I wasn't certain about that one too.

Reply:
> So do you think that changing rgbToRGB to rgbInputToRGBNumbers would be fine without changing the other two.

Yes, actually. The thing is : hextoRGB - it's actually cinverting one color format to other.

rgbToRGB - in this your fucntion you are extracting data for calculation, you are not converting formats.
So htese funcs are alittle different - so names could be alittle different

rgbaToCloseRGB - this looks descriptive to me

Reply:
Oh, I see. I thought we are extracting the RGB values in all the functions. I will make the change then. Thank you.